tests/momento/simple_cache_client/test_dictionary.py

```diff
@@ -287,7 +309,14 @@ def with_bytes_it_succeeds(
         dictionary_field_bytes: bytes,
         dictionary_value_bytes: TDictionaryValue,
     ) -> None:
-        dictionary_setter(client, dictionary_name, dictionary_field_bytes, dictionary_value_bytes, ttl=CollectionTtl())
+        dictionary_setter(
+            client,
+            cache_name,
+            dictionary_name,
+            dictionary_field_bytes,
+            dictionary_value_bytes,
+            ttl=CollectionTtl(),
```

Comment:
Can we omit the ttl from these (here and elsewhere) since the method gives them a default value?

Reply:
It's a setter and setters take a ttl, whether or not the tests currently use them. It's future proofing the interface.

Reply:
Thinking about it, passing in a collectionTtl every time is probably a legacy of the previous way the types were set up. I think I can remove it.

Reply:
Can you do this for list and set as well?

Reply:
I didn't see an issue in any of the other tests. The ttl has to remain in the signature of a collection setter, we just don't need to always pass it in.